addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -5122,4 +5125,51 @@ X[]
             });
         });
     });
+
+    describe('markdown', () => {
```

Comment:
It's not really markdown though.

Reply:
Isn't it?

Reply:
Do we support actual markdown features like `_`, `*` etc ?

Reply:
No but we support backticks :-P If we later support other markdown features we won't have to indent these tests to add a describe markdown ^^

Reply:
I don't think we are going to implement other features but I'm not going to fight to change this describe.